src/core/ui.js

```diff
@@ -234,6 +236,17 @@ export const ui = {
     overlay.addEventListener("click", () => this.closeModal(currentOwner));
     overlay.classList.toggle("respec-show-overlay");
     modal.hidden = false;
+    if (content.tagName === "IFRAME") {
+      modal.insertAdjacentHTML(
```

Comment:
I've an idea - add a footer on iframe one.: a link that asks: "open this UI in new tab"
WDYT @saschanaz ?

Reply:
That sounds good.

Reply:
BTW, I'd prefer to insert the whatever footer in ui/search-xref rather than core/ui.

Reply:
@SuyashSalampuria 
Added a link after iframe with https://github.com/w3c/respec/pull/2712, no need to add invisible button anymore.